index.js

```diff
@@ -522,10 +618,20 @@ const publicApi = {
      * If enabled, the output directory is emptied between
      * each build (to remove old files).
      *
+     * A list of available options can be found at https://github.com/johnagan/clean-webpack-plugin
+     *
+     * For example:
+     *
+     *      Encore.cleanupOutputBeforeBuild(['*.js'], (options) => {
```

Comment:
Maybe we should show the default `['**/*']` as the first option. The issue is that, if I *only* want to specify some options for the plugin... I need to know what this first value is by default, so I can repeat it. That kind of sucks... but I can't think of a good way around it (so showing the default here might make sense)

Reply:
I wanted to but `'**/*'` breaks the multi-line comment :(

Reply:
Ha! What a crazy edge case :p